src/otx/core/data/transform_libs/torchvision.py

```diff
@@ -3508,22 +3508,20 @@ class GenerateTarget(tvt_v2.Transform, NumpytoTVTensorMixin):
             the specific codec for more details.
 
     Args:
-        encoder (dict | list[dict]): The codec config for keypoint encoding.
-            Both single encoder and multiple encoders (given as a list) are
-            supported
-        target_type (str, deprecated): This argument is deprecated and has no
-            effect. Defaults to ``None``
+        input_size (tuple[int, int]): Input image size in [w, h]
```

Comment:
Could you leave TODO comments to change shape order?

Reply:
done https://github.com/openvinotoolkit/training_extensions/pull/3788/commits/9dccf2d0550c5bff84e61d09dd080a4017ff2c0a

Reply:
I changed order from (w, h) to (h, w)